hack/test-go.sh

```diff
@@ -204,7 +207,11 @@ reportCoverageToCoveralls() {
 IFS=',' read -a apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
 for apiVersion in "${apiVersions[@]}"; do
   echo "Running tests for APIVersion: $apiVersion"
-  KUBE_API_VERSION="${apiVersion}" runTests "$@"
+  KUBE_API_VERSION="${apiVersion}"
+  echo "Using standard etcd path prefix: ${ETCD_STANDARD_PREFIX}"
+  ETCD_PREFIX=${ETCD_STANDARD_PREFIX} runTests "$@"
```

Comment:
bash isn't my strong suit, but I think you need to ```KUBE_API_VERSION="${apiVersion}" ETCD_PREFIX=${ETCD_STANDARD_PREFIX} runTests "$@"``` to get the api version set right.